src/ubx.cpp

```diff
@@ -243,6 +243,23 @@ GPSDriverUBX::configure(unsigned &baudrate, const GPSConfig &config)
 			return -1;	// connection and/or baudrate detection failed
 		}
 
+		// Disable GPS protocols at I2C
```

Comment:
Can you do this in `configureDevice` which is used for the newer protocols?

Reply:
Moved under `configureDevice`.